public/app/plugins/panel/logs/LogsPanel.test.tsx

```diff
@@ -145,10 +145,10 @@ beforeAll(() => {
   });
 });
 
-describe('LogsPanel', () => {
+describe.each([false, true])('LogsPanel with controls = %s', (showControls: boolean) => {
```

Comment:
With this we make sure that both modes work in the same way, with and without controls, which just catched a potential regression for Drilldown.